src/main/java/net/torocraft/minecoprocessors/processor/Processor.java

```diff
@@ -540,6 +601,30 @@ void processJnc() {
     }
   }
 
+  void processJg() {
+    if(!carry && !zero) {
+      processJmp();
+    }
+  }
+
+  void processJge() {
+    if((!carry && !zero) || (!carry && zero)) {
```

Comment:
Couldn't this just be:
if(!carry) {
?

Reply:
I was thinking the same thing, but it was hurting my head last night :face_with_head_bandage: